### PR TITLE
[Post] The New Responsive

### LINKED ISSUE #5136: The New Responsive Design [5/18/2021]
* [Draft](https://docs.google.com/document/d/1gb-IuDbzyDhqLik-kWFbjgWZZBCqvurV8D0XNMJawgc/edit?usp=sharing)
* [Discussion](https://groups.google.com/a/google.com/g/chrome-devrel-content/c/38r1R78neZM)

### PR DESCRIPTION
**Pull request by una**
Fixes #5136

(Missing video until tomorrow)

### CODE REVIEW

File: src/site/content/en/blog/new-responsive/index.md [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+}
+
+```
+
+When there is at least `350px` of space in the grid, the card layout goes
+horizontal by being set to `display: flex`, which has a default flex-direction
+of "row". 
+
+With less space, the product cards stack. Each product card styles itself, which
+is something that would be impossible when only able to rely on global styles.
```
Comment (rachelandrew) [inline src/site/content/en/blog/new-responsive/index.md:282]:
```suggestion
With less space, the product cards stack. Each product card styles itself, 
something that would be impossible with global styles alone.

```

File: src/site/content/en/blog/new-responsive/index.md [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+It's combining macro layout with micro layout, and on top of all of that, it's
+taking user customization and form factor into account.
+
+Any of these changes alone would constitute a considerable shift in how we
+design for the web. But combined, they signify a really big shift in how we even
+conceptualize responsive design. It's time to think about responsive design
+beyond viewport size, and start considering all of these new axes for better
+component-based and customized experiences.
+
+> The next era of responsive design is here, and you can already start to
```
Comment (rachelandrew) [inline src/site/content/en/blog/new-responsive/index.md:424]: This seems a bit odd as a quote.

**Review by rachelandrew — APPROVED** [review]
Just a couple of tiny things.

File: src/site/content/en/blog/new-responsive/index.md (hunk 2 above)
Comment (una) [inline src/site/content/en/blog/new-responsive/index.md:425]:
```suggestion
The next era of responsive design is here, and you can already start to
```

**Comment by una** [comment]
Addressed them! I'll add the video link when it goes live, and aim for this to go out tomorrow morning.

**Pull request merged**